xml/Microsoft.Extensions.Http.Logging/IHttpClientAsyncLogger.xml

```diff
@@ -1,148 +1,156 @@
+  <TypeSignature Language="ILAsm" Value=".class public interface auto ansi abstract IHttpClientAsyncLogger implements class Microsoft.Extensions.Http.Logging.IHttpClientLogger" />
+  <TypeSignature Language="DocId" Value="T:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger" />
+  <TypeSignature Language="VB.NET" Value="Public Interface IHttpClientAsyncLogger&#xA;Implements IHttpClientLogger" />
+  <TypeSignature Language="F#" Value="type IHttpClientAsyncLogger = interface&#xA;    interface IHttpClientLogger" />
+  <TypeSignature Language="C++ CLI" Value="public interface class IHttpClientAsyncLogger : Microsoft::Extensions::Http::Logging::IHttpClientLogger" />
+  <AssemblyInfo>
+    <AssemblyName>Microsoft.Extensions.Http</AssemblyName>
+    <AssemblyVersion>8.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Interfaces>
+    <Interface>
+      <InterfaceName>Microsoft.Extensions.Http.Logging.IHttpClientLogger</InterfaceName>
+    </Interface>
+  </Interfaces>
+  <Attributes>
+    <Attribute>
+      <AttributeName Language="C#">[System.Runtime.CompilerServices.NullableContext(1)]</AttributeName>
+      <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.NullableContext(1)&gt;]</AttributeName>
+    </Attribute>
+  </Attributes>
+  <Docs>
+    <summary>An abstraction for asyncronous custom HTTP request logging for a named <see cref="T:System.Net.Http.HttpClient" /> instances returned by <see cref="T:System.Net.Http.IHttpClientFactory" />.</summary>
+    <remarks>
+      <para> Asyncronous methods (such as <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger.LogRequestStartAsync(System.Net.Http.HttpRequestMessage,System.Threading.CancellationToken)" />) would be called from async code paths (such as <see cref="M:System.Net.Http.HttpClient.SendAsync(System.Net.Http.HttpRequestMessage,System.Threading.CancellationToken)" />), and their syncronous counterparts inherited from <see cref="T:Microsoft.Extensions.Http.Logging.IHttpClientLogger" /> (such as <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestStart(System.Net.Http.HttpRequestMessage)" />) would be called from the corresponding sync code paths.
+            </para>
+      <para> It is up to the user implementing the interface to decide where (to <see cref="T:Microsoft.Extensions.Logging.ILogger" />, or anything else) and what exactly to log.
+            However, the implementation should be mindful about potential adverse side effects of accessing some of the <see cref="T:System.Net.Http.HttpRequestMessage" /> or <see cref="T:System.Net.Http.HttpResponseMessage" /> properties, such as reading from a content stream; if possible, such behavior should be avoided.
+            </para>
+      <para> Logging implementation also should not throw any exceptions, as an unhandled exception in logging would fail the request.

```

Comment:
```suggestion
      <para>The logging implementation shouldn't throw any exceptions, as an unhandled exception in logging would fail the request.
```